pkg/jx/cmd/delete_app.go

```diff
@@ -97,16 +99,76 @@ func NewCmdDeleteApp(f Factory, in terminal.FileReader, out terminal.FileWriter,
 
 // Run implements this command
 func (o *DeleteAppOptions) Run() error {
+	var err error
+	var ns string
+	o.jxClient, ns, err = o.Factory.CreateJXClient()
+	o.currentNamespace = ns
```

Comment:
no need to use the temporary `ns` variable here, you can just do `o.jxClient, o.currentNamespace, err = o.Factory.CreateJXClient()`